modules/main-rules/pages/broken-detachments.adoc

```diff
@@ -18,10 +17,12 @@ This is the only type of move you may make with a Broken detachment.
 [[retreat-move]]
 == Retreat move
 
-The Retreat move is a single move in which you may move all of the Broken units up to 20 cm -- regardless of the actual Speed values of units.
+The Retreat move is a single move in which you may move all mobile units in the Broken detachment up to 20 cm -- regardless of their actual Speed values.
 
-* You may make the Retreat move in any direction, subject to the normal rules for terrain effects on movement.
-* If you end a Retreat move such that any Broken units are within 15 cm of *any* enemy unit footnote:[Even Broken enemy units.], then they are *destroyed* -- remove those units as casualties.
+* You may use the Retreat move to move the units in any direction, subject to the normal rules for terrain effects on movement.
+* You may 'move' units 0 cm if you wish.
+Or, it may be that you can't move some units anywayfootnote:[There may be a rule in force that states that you 'may not move' a unit — the Retreat move rules do not override this.].
```

Comment:
Latest iteration of the tweak. Note that I adjusted the dangerous terrain test wording to correlate better with this. https://github.com/GameBrains/er-core/pull/14/files#r684469440

Thus hopefully we don't need to clutter things up with specific mention of the dangerous terrain test or the Heavy Barrage rule just to tie it all together — consistent use of 'may not move' wording should cover it, I think.

Reply:
👍🏼